pkg/resource/replication_group/testdata/test_suite.yaml

```diff
@@ -0,0 +1,23 @@
+tests:
+  -
+    name: "rg_snapshot_cmd"
```

Comment:
any reason for the `_cmd` suffix on the test name here?

Reply:
`cmd` here was to indicate Cluster Mode Disabled.

Reply:
ahhhhhh!